docker-compose.yml

```diff
@@ -104,8 +105,14 @@ services:
         image: nginx:1.15-alpine
         ports:
         - 80:80
+        - 443:443
         volumes:
         - ./volume/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+        - /home/greg/workspace10/OpenELIS/dev/https/greg.dev.openelis.org.crt.pem:/etc/nginx/serverCerts/cert.crt
+        - /home/greg/workspace10/OpenELIS/dev/https/greg.dev.openelis.org.key.pem:/etc/nginx/serverCerts/cert.key
+        - /home/greg/workspace10/OpenELIS/dev/https/passwd:/etc/nginx/private/key_pass
+        extra_hosts:
+        - host.docker.internal:172.20.1.1
         restart: unless-stopped            
```

Comment:
I think the changes in the docker-compose file were also dev specific to your environment , you could just remove them from your PR